### ISSUE TITLE
Incorrect position when rotate keyboard 

### ISSUE THREAD

**Issue by bhumiradixweb** [Mottie/Keyboard#449]
Hello,

I want to rotate keyboard in some case so added following css code:

`.ui-keyboard{transform: rotate(90deg)}`

But it display keyboard in incorrect position. Any solution?

**Comment by Mottie** [comment]
Hi @bhumiradixweb!

Honestly, I wouldn't recommend using that css transform on an entire page. It would be better to let the mobile device OS change the orientation. Basically the problem is that when you use a css `transform: rotate(90deg)`, javascript still returns the original width and height of the element, so when jQuery UI's position utility tries to position the keyboard, it is using the height instead of the rotated width in the calculations.

Anyway, I saw your question on Stackoverflow, so I used the demo you provided there.

I tried fixing it using the [position utility](http://api.jqueryui.com/position/) built-in offsets (e.g. `at2: 'center-90 bottom'`), but it doesn't appear to work for "center" locations, at least not when an element is transformed. So the only solution I could think of would be to adjust it manually ([demo](http://jsfiddle.net/88zg4n3e/4/)):

```js
 $('#test').keyboard({
   layout: 'qwerty',
   usePreview: false,
   beforeVisible: function(event, kb) {
     // using setTimeout because jQuery UI position is
     // called after beforeVisible; and visible is called
     // 200ms after
     setTimeout(function() {
       var left = parseFloat(kb.$keyboard.css('left')) + 90;
       kb.$keyboard.css('left', left);
     }, 10);
   }
 });
```

I hope that helps!
